index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav class="menu-wrap">
+      <div class="logo">
+        <span class="brand">Saw Mon Han</span>
+        <input type="checkbox" class="toggler" />
+        <div class="hamburger"><div></div></div>
```

Comment:
- [ ] Kindly, It would be better also for your hamburger button to have a ```<nav>``` tag. 😎😎
  This will be for semantics reasons and the right position for convention and user experience for mobile devices.

Remember also, the [```<nav>``` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) in HTML represents a section of a page whose purpose is to provide navigation links, either within the current document or to other documents. 

Common examples of navigation sections are **menus, tables of contents, and indexes.** ✨✨